client/src/main/java/com/sendwithus/model/ActivatedDrip.java

```diff
@@ -9,11 +9,11 @@
     private String locale; // must match JSON response
     private String message; // must match JSON response
 
-    public DripCampaign getDrip_campaign() {
+    public DripCampaign getDripCampaign() {
```

Comment:
Yup, ActivatedDrip and DeactivatedDrip are both new classes introduced by Yousef.  Nobody should actually be using them yet though, as we haven't published a new maven release